website/blog/2023-11-11-AgentEval/index.md

```diff
@@ -0,0 +1,141 @@
+
+---
+title:  AgentEval: Assessing Task Utility for LLM-powered Applications
+authors: julianakiseleva, Narabzad
+tags: [LLM, GPT, evaluation, task utility]
+---
+
+![Fig.1: An verification Framework](img/agenteval-CQ.png)
+
+**TL;DR:**
+* We introduce `AgentEval` — the first version of the framework to automatically assess task utility for an arbitrary application. It suggests criteria to explain task utility and then quantifies these criteria for logs of your system.

```

Comment:
This sentence is too complex. Simplify

Reply:
@gagb can you please elaborate what sound too complex?

Reply:
Eg
what does "task utility" means. And then it unclear what "automatically assess task utility" could mean. Or what "explain task utility" could mean. If this is a new term that the article is proposing, maybe emphasize it and say "described below".
what is the scope of "system"

Reply:
Blog introduces those concepts  -- now TLDR states which concepts are being introduced.

Reply:
The new version of the blog addresses an issue you mentioned above.